testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -454,7 +454,7 @@
                  'third A'           => 'bind#available_zones#2#records#A#add_item',
                  'third NS'          => 'bind#available_zones#2#records#NS#@#add_item',
                  'partitions'        => 'partitioning#0#partitions#add_item' }
-  find(:xpath, "//button[@id='#{sectionids[section]}']").click
+  find(:xpath, "//i[@id='#{sectionids[section]}']").click
```

Comment:
Hi @mateiw . Is this complete?

I mean, apparently `dhcpd#hosts#add_item` changed into `dhcpd#subnets#0#hosts#add_item` (see recent failures in head).

Or am I referring other formula changes than yours?

Reply:
Forget it, your change should be enough :smile_cat: . We have both global dhcpd host settings and subnet-specific ones.